apps/frontpage/components/docs/mdx/home-renderers.tsx

```diff
@@ -40,6 +40,12 @@ export function HomeRenderers({ activeVersion }: HomeRenderersProps) {
         title="React Native"
         subtitle="on device"
       />
+      <Card
```

Comment:
The original PR that added this placed it at the end of the card grid. Was it moved up on purpose?

I think it's more appropriately placed at the end.